setup.py

```diff
@@ -36,7 +36,8 @@
           'tzlocal>=2.0,<4.1',
           'slackclient>=2.7,<2.10',
           'psutil==5.8.0',
-          'ujson>=4.1,<4.3'
+          'ujson>=4.1,<4.3',
+          'dnspython>=2.0,<2.2',
```

Comment:
can you pin this to a specific minor version, eg `2.1.*`, it's more deterministic.

Reply:
Should we do the same in `pipelinewise-tap-mongodb`? This requirement is a copy-paste from there: https://github.com/transferwise/pipelinewise-tap-mongodb/blob/master/setup.py#L21

Reply:
yep